assets/js/admin.js

```diff
@@ -24,66 +24,48 @@
 
 	//select2
 	$(document).ready(function() {
-		$('.btm-callback-action-settings').select2();
-		$('.btm-status-settings').select2();
-		$('.btm-users-settings').select2({
-			placeholder: 'Select users'
-		});
-	});
 
-	//custom bulk
-	$( '.btm-bulk-delete' ).click( function(){
-		if($(this).prop("checked") === true){
-			$( '.btm-delete' ).prop("checked", true);
+		if( $('#failed').attr('checked') ){
```

Comment:
@eriknovembit I'm not CSS guru, but I think working with ID `#failed` can bring up unexpected results ))

Reply:
I think you right. I fixed them